### DISCUSSION TITLE
Rebase around OpenSUSE Leap/Slowroll

### DISCUSSION THREAD

**Discussion by vyvir** [elementary/triage#439] [Ideas]
Greetings,

Since it's the perfect time to share your vision for the next major version of elementary OS, I would like to share mine, too.
I think ElementaryOS would greatly benefit from rebasing around either OpenSUSE Leap or Slowroll.
Reasons for this change:
- Stable base: Leap is comparable to Ubuntu LTS
- No snap packages; Firefox is packaged as an RPM. Both ESR and regular versions are available.
- Official NVIDIA repo
- Flagship Secure Boot support
- Support for XFS and BTRFS out of the box
- Snapshots are enabled by default and allow the user to rollback in case something goes awry (it is better to split the root partition from the home partition for this use case) 

There are [already community efforts to package Pantheon for OpenSUSE](https://build.opensuse.org/project/show/X11:Pantheon:Branch). While efforts have stalled since 2022, the repo still provides a base to work from. Let me know what you think!

**Comment by vyvir** [comment]
Here's a similar idea: https://www.reddit.com/r/elementaryos/comments/17n71f0/comment/k8n1jxa/

**Comment by OulipianSummer** [comment]
I think the OpenSUSE official installer supports using Pantheon as a DE, why not use that instead? Rebasibg the entire package set around OpenSUSE would be a huge undertaking, and someone else in the Linux universe is already doing this.
